### PR TITLE
IPv6 support and JS based IPv4/IPv6 input validation #2

### PR DESCRIPTION
**Pull request by mrizvic**

### CODE REVIEW

**Comment by karlitschek** [comment]
@rullzer Can you have a look?

**Comment by mrizvic** [comment]
I regularly do the "update branch" but today I noticed that the ci/drone/pr check fails because:
```
Generating autoload files
+ cd ../server
+ php occ app:enable limit_login_to_ip
App "Restrict login to IP addresses" cannot be installed because it is not compatible with this version of the server.
```

**Comment by rullzer** [comment]
somehow this never showed up on my notifications. let me have a look later today and then get it in.

**Comment by mrizvic** [comment]
> somehow this never showed up on my notifications. let me have a look later today and then get it in.
Looks like you guys are really busy. Kudos to nextcloud staff! Anything I can do to help out? I'd like to get involved in a project like this.

File: js/settings.js [hunk 1]
```diff
@@ -108,6 +108,18 @@
...
+                var r4 = new RegExp(regexFilter).exec(ipAddress);
+                var r6 = new RegExp(regexFilter).exec(ipAddress);
```
Comment (rullzer) [inline js/settings.js:116]: You are using the same filter here... executing once should be enough then

Comment (mrizvic) [reply js/settings.js]: fixed

**Comment by rullzer** [comment]
Could you rebase and squash?
Also please sign off your commits (s e the dco check)

**Comment by rullzer** [comment]
Merged in https://github.com/nextcloud/limit_login_to_ip/pull/46

**Pull request closed without merging**